serverless/src/shared/municipalities/index.js

```diff
@@ -153,6 +153,35 @@ const getMunicipalitiesOfUser = userId => {
   return ddb.query(params).promise();
 };
 
+// Get all municipalities of user. But not just the ags, but also
+// all the data (name, slug, population)
+const getMunicipalitiesOfUserWithData = async userId => {
+  const municipalities = [];
+
+  // Get municipalities for which the user has signed up for
+  const { Count, Items } = await getMunicipalitiesOfUser(userId);
+
+  if (Count !== 0) {
+    // Get municipality names for all municipalities
+    await Promise.all(
+      Items.map(async municipality => {
+        const municipalityResult = await getMunicipality(municipality.ags);
+
+        // Municipality should be definitely there, but we'll check anyway
+        if ('Item' in municipalityResult) {
+          municipalities.push({
+            ...municipality,
+            name: municipalityResult.Item.name,
```

Comment:
Ah cool, du hängst mir gleich die Namen an, merci!

Reply:
Ja, das war schon so im users/me Endpunkt. Da hatten wir es gebraucht. Hab nur die Funktion von dort abstrahiert, um sie auch in users/{userId} zu verwenden.